tera-templates/docsrs/Cargo.toml.example

```diff
@@ -0,0 +1,44 @@
+{# The example Cargo.toml used in about.html #}
```

Comment:
Why do all the templates have `docsrs/` as a leading prefix? We know they must be for docs.rs because that's the repository they're in.

Reply:
They’re templates that pertain to the core of the site

Reply:
As opposed to what? Aren't all the templates part of docs.rs?

Reply:
As opposed to user controlled (so to speak) and content pages, these are internal pages

Reply:
I don't think `docsrs` is a good name then. If you want to emphasize that they're internal pages you could use `internal/` or `core` as a prefix but I don't think they need a prefix at all really.